src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py

```diff
@@ -81,6 +81,11 @@
           text: az storage container create -n MyStorageContainer --fail-on-exist
 """
 
+helps['storage account create'] = """
```

Comment:
I'd have to define the string in the docstring of the 2 methods for both API versions.